connector/connect/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -141,11 +141,14 @@ message Expression {
     // (Optional) Function arguments. Empty arguments are allowed.
     repeated Expression arguments = 2;
 
+    // (Required) Indicate if this function should be applied on distinct values.
+    bool is_distinct = 3;
```

Comment:
Do we need a negative test case here where for example if I call `substr` but with this `is_distinct=true`?